app/cmd/computer_launch.go

```diff
@@ -213,7 +213,7 @@ jcli agent launch agent-name --type jnlp`,
 		}
 
 		var f *os.File
-		if f, err = ioutil.TempFile("/tmp", "agent.jar"); err == nil {
+		if f, err = ioutil.TempFile("", "agent.jar"); err == nil {
```

Comment:
I guess this change should not appear here.

Reply:
I was also wondering why these changes appear here cuz I hadn't even opened these files and I pulled from origin latest. 
![image](https://user-images.githubusercontent.com/53268258/134925707-affb4f47-5577-4894-85a0-cea85f2e7697.png)

Reply:
maybe just misused of git command

Reply:
Thx. Working on this.